src/S3/S3ClientInterface.php

```diff
@@ -48,6 +48,17 @@ public function getObjectUrl($bucket, $key);
      */
     public function doesBucketExist($bucket);
 
+    /**
+     * Determines whether or not a bucket exists by name.
+     *
+     * @param string $bucket  The name of the bucket
+     * @param bool $accept403 Changes behavior to return true in the case of a 403.
```

Comment:
How about "set to true to have this method return true in the case of a 403 credential error; using the default value of false will cause the error to be thrown instead"